adapter/config/crd/client.go

```diff
@@ -194,17 +194,17 @@ func (cl *Client) RegisterResources() error {
 				}
 			}
 			glog.V(2).Infof("missing status condition for %q", name)
-			return false, err
+			return false, nil
 		}
```

Comment:
On a phone so hard to see the big picture but why would returning less info (nil vs err) be better ?

Reply:
`err` is nil here always since it's a wrong variable.